ecs-init/docker/docker.go

```diff
@@ -393,6 +404,10 @@ func (c *Client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 	}
 
 	binds = append(binds, getDockerPluginDirBinds()...)
+
+	// only add bind mounts when the src file/directory exists on host; otherwise docker API create an empty directory on host
+	binds = append(binds, getCapabilityExecBinds(pathExistsAndValid)...)
```

Comment:
is there a unit test that needs to consider this?

Reply:
yeah, we should add on to this https://github.com/aws/amazon-ecs-init/blob/b5d2758a0ba113cd06d5e9867e15c49c206d3207/ecs-init/docker/docker_test.go#L32

Reply:
don't think it'd be worthwhile to only test against `append()`, so I'll update the current `TestGetCapabilityExecBinds` test a bit to include this behavior

Reply:
I didn't meat to test that append works, I mean to update the relevant tests for the function you're adding behavior to. As Sharanya points out, some tests are expecting 19 bind mounts, this means that probably we're expecting 20 now, under certain conditions... that's what we should test

Reply:
got it!